sentry-expo-migration.md

```diff
@@ -28,6 +28,15 @@ On your terminal, run the following commands to remove `sentry-expo` and install
 - Replace any usage of `Sentry.Native.*` with `Sentry.*`, where `Sentry` is imported from `@sentry/react-native` as above (e.g. `Sentry.Native.captureException` -> `Sentry.captureException`)
 - Replace any usage of `Sentry.Browser.*` with `Sentry.*`, where `Sentry` is imported from `@sentry/browser`.
 
+## Wrap your App with `Sentry.wrap()`:
+```javascript
+import * as Sentry from '@sentry/react-native';
+
+// Your App component here
+
+export default Sentry.wrap(App);
+```
```

Comment:
````suggestion
## Wrap your App with `Sentry.wrap()`

```js
import * as Sentry from '@sentry/react-native';

// Your App component here

export default Sentry.wrap(App);
```
````

Removing colon at the end of the section title for consistency across all the guides. As per our writing style guide, we do not use it.